test/JDBC/expected/BABEL-3166-before-14_11-or-15_6-vu-verify.out

```diff
@@ -0,0 +1,38 @@
+-- Look at function's probin for typmod information
+SELECT proname, probin FROM pg_proc WHERE proname = 'babel_3166_func';
```

Comment:
add a few tests for declared param,
declare @a binary = <val>
select @a 

declare @a binary(10) = <val where val length > or <= 10>
select @a


also , add tests for typmod cast,
declare @a binary = <val>
select cast(@a  as binary(100)) or binary(max) 

same for implicit casting as well as by retrieving data from table